lms/djangoapps/mobile_api/users/views.py

```diff
@@ -122,6 +125,13 @@ class UserCourseStatus(views.APIView):
 
     http_method_names = ["get", "patch"]
 
+    def dispatch(self, request, *args, **kwargs):
+        if request.method == 'PATCH':
```

Comment:
For safety, might want to also guard PUT and DELETE and such. You can do something like:

```
from rest_framework.permissions import SAFE_METHODS
if request.method in SAFE_METHODS:
  dispatch()
else:
  with atomic: dispatch()
```